### PR TITLE
Update drawer component

### PR DESCRIPTION
**Pull request by yurii-ve**
Fixes https://github.com/wixplosives/codux/issues/23208

- used E-Commerce ReClaim implementation of the drawer component (cart title is part of the CartView component not Drawer)
- removed redundant boards

### CODE REVIEW

File: src/components/drawer/drawer.tsx [hunk 1]
```diff
@@ -1,51 +1,63 @@
...
+        <ClientOnly>
+            {() =>
+                createPortal(
+                    <AnimatePresence>
+                        {open && (
+                            <>
+                                <motion.div
+                                    className={styles.backdrop}
+                                    initial={{ opacity: 0 }}
+                                    animate={{ opacity: 1 }}
```
Comment (olehrakwix) [inline src/components/drawer/drawer.tsx:37]: so it is now not arriving from the right but but eases-in?

**Review by olehrakwix — APPROVED** [review]

File: src/components/drawer/drawer.tsx (hunk 1 above)
Comment (yurii-ve) [reply src/components/drawer/drawer.tsx:37]: it's arriving from the right... just with improved animation :)

File: src/components/drawer/drawer.tsx [hunk 2]
```diff
@@ -1,51 +1,63 @@
...
+import { ClientOnly } from 'remix-utils/client-only';
+import classNames from 'classnames';
+import { AnimatePresence, motion } from 'framer-motion';
+import { RemoveScroll } from 'react-remove-scroll';
 import styles from './drawer.module.scss';
 
-export interface DrawerProps {
-    className?: string;
+interface DrawerProps {
+    open: boolean;
```
Comment (VladyslavG) [inline src/components/drawer/drawer.tsx:10]: why did you change this prop name ?

Comment (yurii-ve) [reply src/components/drawer/drawer.tsx:10]: it's copied from the Drawer of the ReClaim template

**Review by VladyslavG — APPROVED** [review]
